incubator/hnc/Makefile

```diff
@@ -107,7 +111,7 @@ clean: krew-uninstall
 
 # Installs the Linux kubectl plugin to $GOPATH/bin, assume that this is in your PATH already.
 kubectl: build
-	cp bin/kubectl/kubectl-hns_linux_amd64 ${GOPATH}/bin/kubectl-hns
+	cp bin/kubectl/kubectl-hns_$(OS_NAME)_amd64 $(GOBIN)/kubectl-hns
```

Comment:
Does GOBIN reliably default to GOPATH/bin if it's not explicitly set? If not, should we set it ourselves?

Reply:
Yes. the `go env GOPATH` resolves to `<user-home-dir>/go` if GOPATH is unset. So, no need to set this explicitly.

Reply:
Right, but this change uses `GOBIN`, not `GOPATH`. I double-checked and GOBIN is unset on my workstation, so this line will fail.

A more robust solution might be to say something like `GOBIN ?= $(shell go env GOPATH)/bin` which will set it to the default if it's unset.

Reply:
@adrianludwin - I believe that is how GOBIN is being set in the Makefile at the moment. https://github.com/kubernetes-sigs/multi-tenancy/blob/56839e8dbf3e8bdb676f7d1a86e414efab6850fc/incubator/hnc/Makefile#L79

And we are using the same variable here? Should work right?

Reply:
Whoops I forgot we already had that! Ok that lgtm

Reply:
Sure. Thanks for the review.